content/docs-v1/install-uninstall.md

```diff
@@ -7,181 +7,72 @@ aliases:
   - /documentation/install-update/
 ---
 
-{{< q-class "box warning" >}}
-- Quire is in a limited beta, © J. Paul Getty Trust, and not yet released as open-source software. For a free license to use Quire for your publication projects (and for additional access to support) [please sign up](https://goo.gl/forms/Ih4SOHje0Z9dm3C73).
-{{< /q-class >}}
-
-## macOS Installation
-
-Installing and running Quire requires using the Terminal {{< q-def "command-line shell" >}}. Open it from your Applications/Utilities folder or by pressing Command–Space Bar and typing “Terminal”. If you are new to the command-line, read our tutorial [*Working in a Command-line Shell*](/learn/tutorial/#1-working-in-a-command-line-shell).
+There is no user interface for Quire so it may look or feel different than other publishing software. This was an intentional decision. The transparency of Quire's backend provides more flexibility and opportunities for customization.
 
-### Quick Install
+When you install Quire what you are actually installing is something called a {{< q-def "command-line interface" >}} or CLI. The CLI enables you to create, preview and output publications using simple commands like `quire new`, `quire preview` and `quire build`.
 
-If you’re eager to get started, this will install the complete Quire package but without e-book or PDF output capability. These may be added later, by following steps 3–5 in the Full Install guidelines below.
+Quire is completely free and open source. Installation instructions can be found below.
```

Comment:
Should we create a glossary term for "open source" and/or link to the actual open source license?

Reply:
I like the idea of linking to the license. Where can I find it? I don't see it in the repo anywhere, and can't remember the actual CC type we are going with.

Reply:
The OS license we're using is the [3-clause BSD](https://opensource.org/licenses/BSD-3-Clause). I just submitted a PR for it to be added to the quire repo. Tomorrow, when the `main` branch is updated from `feature/11ty`, the OS license will be at https://github.com/thegetty/quire/blob/main/LICENSE.